frontend/src/pages/MessagingQueues/MQDetails/MQTables/MQTables.tsx

```diff
@@ -177,8 +188,35 @@ function MessagingQueuesTable({
 		{
 			onSuccess: (data) => {
 				if (data.payload) {
-					setColumns(getColumns(data?.payload, history));
-					setTableData(getTableData(data?.payload));
+					if (
+						type === 'Overview' &&
+						selectedView === MessagingQueuesViewType.producerLatency.value &&
+						tableApiPayload?.detailType === 'producer'
+					) {
+						setColumns(
+							getColumnsForProducerLatencyOverview(
+								(data?.payload as TopicThroughputProducerOverviewResponse['payload'])
+									.result[0].list,
```

Comment:
will `result[0].list` always be present ?

Reply:
changed